public_html/wp-content/plugins/wc-post-types/wc-post-types.php

```diff
@@ -654,9 +654,9 @@ public function get_speaker_wporg_permalink( $speaker_id ) {
 	/**
 	 * Returns an anchor permalink for a Speaker or Session
 	 *
-	 * Any page with the [speakers | sessions] shortcode will contain IDs that can be used as anchors.
+	 * Any page with the Speakers or Sessions block will contain IDs that can be used as anchors.
 	 *
-	 * If the current page contains the corresponding shortcode, we'll assume the user wants to link there.
+	 * If the current page contains the corresponding block, we'll assume the user wants to link there.
 	 * Otherwise, we'll attempt to find another page that contains the shortcode.
```

Comment:
- [ ] Change `shortcode` to `block` here too
- [ ] `[MIGHT]` It might be worth explaining the feature flag in a doc comment here, as a reminder to future devs that these blocks used to be shortcodes.